index.js

```diff
@@ -45,6 +47,7 @@ class LedgerBridgeKeyring extends EventEmitter {
     this.hdPath = opts.hdPath || hdPathString
     this.bridgeUrl = opts.bridgeUrl || BRIDGE_URL
     this.accounts = opts.accounts || []
+    this.accountIndexes = opts.accountIndexes || {}
```

Comment:
I think we can drop the fallback here:

```suggestion
    this.accountIndexes = opts.accountIndexes
```

Reply:
Can we? If `this.accountIndexes` is undefined, then various other lines added in this PR will fail.

Reply:
Good point, this suggestion doesn't account for deserializing an old instance